src/api-reference/request/get-started.markdown

```diff
@@ -0,0 +1,173 @@
+---
+title: Get Started
+layout: reference
+---
+
+# Request
+Concur Request automates the spend request and approval process for both travel and everyday expenses, giving you the data you need to accurately track and better control spending. By increasing visibility into planned expenses and up-to-date budget data, you can make strategic spending decisions before any spending actually occurs. The Request resource provides many abilities, including requests creation and transition into the approval workflow.
+
+## Get Started
+
+- [Overview](#overview)
+- [Version](#version)
+  - [Overview version V4.0](#overviewV4)
+- [Regional Availability](#regional)
```

Comment:
_note: when clicking on the button Display the Rich Diff, the link Regional Availability does not display. Might just be a question of display in Github but to keep in mind for information_

Reply:
FIXED